bundles/org.openhab.binding.yeelight/README.md

```diff
@@ -4,11 +4,11 @@ This binding integrates the [Yeelight Lighting Product](https://www.yeelight.com
 
 ## Supported Things
 
-- [Yeelight LED White Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-w) (Thing type `dolphin`)
```

Comment:
Why do you remove references to dolphin and wonder thing types?  They are apparently still supported by the binding, at least I can find the XML description. 

Regarding the WEB links, maybe you can replace them rather than removing them?

Reply:
Probably here is a confusion between Thing type and device type. I was trying to reflect all supported device types:
public enum DeviceType {
    mono,
    mono4,
    ct_bulb,
    color,
    color4,
    ceiling,
    ceiling1,
    ceiling3,
    ceiling4,
    ceil26,
    ceiling11,
    stripe,
    strip6,
    desklamp
}

Does it looks better now? Regarding URLs, I'm not sure what device id correlates to which URL exactly.